### PR TITLE
BWS-55: * bug fix

### PR DESCRIPTION
**Pull request by bohdanshcherba**

### CODE REVIEW

File: backend/src/data/repositories/group/group.repository.ts [hunk 1]
```diff
@@ -64,14 +64,16 @@ class Group {
...
     });
-    await this.#UsersGroupsModel.query().insert(
-      workersIds.map((workerId) => ({
-        id: getRandomId(),
-        userId: workerId,
-        groupId: id,
-        createdAt: createdAt,
-      })),
-    );
+    if (workersIds.length !== 0) {
```
Comment (what1s1ove) [inline backend/src/data/repositories/group/group.repository.ts]:
```suggestion
   const hasWorkersIds = Boolean(workersIds.length )
    if (hasWorkersIds) {
```

File: shared/src/validation/validation-schemas.ts [hunk 2]
```diff
@@ -1,3 +1,3 @@
 export { EamWorkerCreate } from './eam-worker/worker';
 export { eamMasterSignUp, eamMasterSignIn } from './eam-master/eam-master';
-export { eamGroupCreate } from './eam-group/eam-group';
+export { eamGroupCreate, eamGroupConfigurate } from './eam-group/eam-group';
```
Comment (what1s1ove) [inline shared/src/validation/validation-schemas.ts:3]:
```suggestion
export { eamGroupCreate, eamGroupConfigurate } from './eam-group/eam-group';
```
what are the differences between eamGroupCreate and eamGroupConfigurate?

File: shared/src/validation/eam-group/eam-group-create/group-create.validation-schema.ts [hunk 3]
```diff
@@ -21,7 +21,7 @@ const eamGroupCreate = Joi.object({
   [getNameOf<EAMGroupCreateRequestDto>('tenantId')]: Joi.string()
     .trim()
     .required(),
-  [getNameOf<EAMGroupCreateRequestDto>('workersIds')]: Joi.array().required(),
+  [getNameOf<EAMGroupCreateRequestDto>('workersIds')]: Joi.array(),
```
Comment (what1s1ove) [inline shared/src/validation/eam-group/eam-group-create/group-create.validation-schema.ts:24]: the length of array must be at least 1

File: shared/src/validation/eam-group/eam-configurate-group/eam-configurate-group.validation-schema.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    .min(EAMGroupValidationRule.NAME_MIN_LENGTH)
+    .max(EAMGroupValidationRule.NAME_MAX_LENGTH)
+    .regex(EAMGroupValidationRule.NAME_REGEX)
+    .required()
+    .messages({
+      'string.empty': EAMGroupValidationMessage.NAME_REQUIRE,
+      'string.min': EAMGroupValidationMessage.NAME_MIN_LENGTH,
+      'string.pattern.base': EAMGroupValidationMessage.NAME_REGEX,
+    }),
+  [getNameOf<EAMGroupCreateRequestDto>('workersIds')]: Joi.array(),
```
Comment (what1s1ove) [inline shared/src/validation/eam-group/eam-configurate-group/eam-configurate-group.validation-schema.ts:21]: the length of array must be at least 1

File: frontend/src/components/eam-configurate-group/helpers/get-rows.helper.ts [hunk 5]
```diff
@@ -3,15 +3,25 @@ import { EAMWorkerGetAllItemResponseDto } from 'common/types/types';
...
 const getRows = (workers: EAMWorkerGetAllItemResponseDto[]): Row[] => {
   return workers.map((item: EAMWorkerGetAllItemResponseDto) => {
-    const { id, name } = item;
+    const { id, name, groups, createdAt } = item;
+
+    const groupsName = groups.map((item) => item.name).join(', ');
+
+    const reDate = /(\d{4})-(\d{2})-(\d{2})T(\d{2}):(\d{2}):(\d{2})/;
+    const el = reDate.exec(createdAt);
+    const date = `${el![1]}/${el![2]}/${el![3]} ${el![4]}:${el![5]}`;
```
Comment (what1s1ove) [inline frontend/src/components/eam-configurate-group/helpers/get-rows.helper.ts]:
```suggestion
```
don't need to pretify date for now

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: shared/src/validation/eam-group/eam-configurate-group/eam-configurate-group.validation-schema.ts (hunk 4 above)
Comment (bohdanshcherba) [reply shared/src/validation/eam-group/eam-configurate-group/eam-configurate-group.validation-schema.ts:21]: User can create a group with no users in it

Comment (what1s1ove) [reply shared/src/validation/eam-group/eam-configurate-group/eam-configurate-group.validation-schema.ts:21]: oh sorry, you're right!

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**